### ISSUE TITLE
[1.15.2] Server kicks you 

### ISSUE THREAD

**Issue by BedrockLegends** [PrinceOfAmber/Cyclic#1401]
Minecraft Version: 1.15.2

Forge Version:  31.1.50

Mod Version: 0.4.0

Single Player or Server: server

Describe problem (what you were doing; what happened; what should have happened):
Logging into a new world

Join a server and boom error in on screenshot
![2020-04-26_19 07 46](https://user-images.githubusercontent.com/48525410/80323511-6afc2900-87f1-11ea-89a3-adeee7cd8ef7.png)

Log file link:

Video/images/gifs (direct upload or link): 

If your bug is related to anything causing LAG you need to measure the lag by taking a sample https://minecraft.curseforge.com/projects/sampler

**Comment by Lothrazar** [comment]
any log files showing where that could have came from?

**Comment by BedrockLegends** [comment]
@Lothrazar I would have to re add the mod to the pack to get the log as I removed the mod as even going to a lower version still caused it

**Comment by tobi-polar** [comment]
I'm not sure whether to open a new issue or not but my pack is experiencing something similar, some people (Not everyone, I don't, but alot of people do,) get an issue connecting to my server. I made a test server, all mod versions are the same, except cyclic is removed. They're able to join the test server.

I then went a step further and removed it from the main server, they can now join fine.

I don't know if removing it was a placebo or if it's actually the problem.

Here's their latest.log trying to join the server WITH cyclic
https://cdn.discordapp.com/attachments/704645831164166285/707441640574419044/latest.log

Here's their log without it
https://cdn.discordapp.com/attachments/704645831164166285/707449229823442944/latest.log

@Lothrazar
